tests/operators/test_agnostic_stats_check.py

```diff
@@ -316,3 +316,20 @@ def test_stats_check_postgres_bigQuery_not_exists(self):
             assert True
         except ValueError as e:
             assert False
+
+    def test_stats_check_on_tables_on_different_db(self):
```

Comment:
@utkarsharma2 it may be worth using `pytest.assert` to check if exceptions are raised:
https://docs.pytest.org/en/6.2.x/assert.html#assertions-about-expected-exceptions

This also applies to the other tests which were added as part of this PR.

Reply:
@tatiana added this. if tests pass I'll merge.